src/common/transformations/src/transformations/flush_fp32_subnormals_to_zero.cpp

```diff
@@ -36,14 +36,28 @@ ov::pass::FlushFP32SubnormalsToZero::FlushFP32SubnormalsToZero() {
         bool has_subnormals = false;
         for (size_t i = 0; i < size; ++i) {
             if (fpclassify(std::abs(data[i])) == FP_SUBNORMAL) {
-                data[i] = 0.0f;
                 has_subnormals = true;
+                break;
             }
         }
-        if (has_subnormals)
-            return true;
+        if (!has_subnormals)
+            return false;
+
+        auto new_constant = std::make_shared<ov::opset8::Constant>(ov::element::f32, node->get_shape());
+        auto* dst_data = const_cast<float*>(new_constant->get_data_ptr<float>());
```

Comment:
it's ok for this PR, but it is not recommended to use `const_cast`
I believe it's better to create ov::Tensor and use the corresponding ctor of Constant